runtime/lua/vim/_defaults.lua

```diff
@@ -67,6 +67,21 @@ do
   --- See |&-default|
   vim.keymap.set('n', '&', ':&&<CR>', { desc = ':help &-default' })
 
+  --- Use Q in visual mode to execute a macro on each line of the selection. #21422
+  ---
+  --- Applies to @x and includes @@ too.
+  vim.keymap.set(
+    'v',
```

Comment:
```suggestion
    'x',
```

"v" maps both Visual _and_ Select mode. To map Visual mode only, use "x".

Mappings with `lhs`es starting with printable characters ought not to be defined for Select mode, as they're expected to replace the selection and enter Insert mode.

Reply:
Oh, on it.